i-d/pdns-qof.xml

```diff
@@ -482,11 +471,68 @@ ws              = *(
       </reference>
     </references>
 
+    <references>
+      <name>Informative References</name>
+      &RFC7871;
+
+      <reference anchor="BAILIWICK"
+      target="https://archive.farsightsecurity.com/Passive_DNS/passive_dns_hardening_handout.pdf">
+      <front>
+        <title>Passive DNS Hardening</title>
+        <author fullname="Robert Edmonds" />
+        <date year="2010" />
+      </front>
+    </reference>
 
-    <references title="Informative References">
-      <!-- Here we use entities that we defined at the beginning. -->
-      <!-- &I-D.narten-iana-considerations-rfc2434bis; -->
-      <!-- &I-D.draft-bortzmeyer-dnsop-dns-privacy; -->
+      <reference anchor="PDNSCLIENT" target="https://github.com/chrislee35/passivedns-client">
+        <front>
+          <title>Queries 5 major Passive DNS databases: BFK, CERTEE, DNSParse, ISC, and VirusTotal.</title>
+          <author fullname="Chris Lee" />
+          <date year="2013" />
+        </front>
+      </reference>
+
+      <reference anchor="REST"
+      target="http://www.ics.uci.edu/~fielding/pubs/dissertation/rest_arch_style.htm">
+      <front>
+        <title>Representational State Transfer (REST)</title>
+        <author fullname="Roy Thomas Fielding" />
+        <date year="2000" />
+      </front>
+    </reference>
+
+    <reference anchor="DNSDB" target="https://api.dnsdb.info/">
+      <front>
+        <title>DNSDB API</title>
+        <author fullname="Farsight Security" />
+        <date year="2013" />
+      </front>
+    </reference>
+
+    <reference anchor="PDNSCERTAT"
+      target="http://www.centr.org/system/files/agenda/attachment/d4-papst-passive_dns.pdf">
```

Comment:
That link seems broken!

Reply:
If we cannot get the PDF back, we can safely remove the reference. As discussed, I checked on archive.org and there is archive of this presentation.